tests/tests/test-tx-index-bound-check.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from "chai";
+import { createTransfer } from "../util/transactions";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+
+describeDevMoonbeam("Transaction Index", (context) => {
+  it("should get transaction by index", async function () {
+    const block = 1;
+    const index = 0;
+    const testAccount = "0x1111111111111111111111111111111111111111";
+    await context.createBlock({
```

Comment:
This should be moved in a "before" as it mutate the state of the chain being also tested in the next `it` test.